src/loadgenerator/locustfile.py

```diff
@@ -129,3 +131,33 @@ def on_start(self):
         ctx = baggage.set_baggage("synthetic_request", "true")
         context.attach(ctx)
         self.index()
+
+
+browser_traffic_enabled = os.environ.get('LOCUST_BROWSER_TRAFFIC_ENABLED', False)
+
+if browser_traffic_enabled:
+    class WebsiteBrowserUser(PlaywrightUser):
+        headless = True  # to use a headless browser, without a GUI
+        multiplier = 1  # run concurrent playwright sessions/browsers for each Locust user
+        wait_time = between(1, 10)
+
+        @task
+        @pw
+        async def open_cart_browser_page(self, page: PageWithRetry):
+            try:
+                async with event(self, "Load up Cart Page"):
+                    await page.goto("/cart")
+            except:
+                pass
+
+        @task
+        @pw
+        async def open_home_browser_page(self, page: PageWithRetry):
+            try:
+                async with event(self, "Load up Astronomy Shop home page"):
+                    await page.goto("/")
+                async with event(self, "Click on a button"):
```

Comment:
Should this be `Click on Go Shopping` button?

Reply:
This is just an event being logged, which does not produce any error. However, I do agree that Click on Go shopping button would be a better naming :)

Reply:
https://github.com/open-telemetry/opentelemetry-demo/pull/1266/commits/45e0a4ef80d0ee9149d4812420e8a41837396097